verticapy/tests_new/plotting/highcharts/machine_learning/test_highcharts_stepwise.py

```diff
@@ -0,0 +1,139 @@
+"""
+(c)  Copyright  [2018-2023]  OpenText  or one of its
+affiliates.  Licensed  under  the   Apache  License,
+Version 2.0 (the  "License"); You  may  not use this
+file except in compliance with the License.
+
+You may obtain a copy of the License at:
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless  required  by applicable  law or  agreed to in
+writing, software  distributed  under the  License is
+distributed on an  "AS IS" BASIS,  WITHOUT WARRANTIES
+OR CONDITIONS OF ANY KIND, either express or implied.
+See the  License for the specific  language governing
+permissions and limitations under the License.
+"""
+# Pytest
+import pytest
+
+# Standard Python Modules
+
+
+# Other Modules
+
+
+# Verticapy
+from verticapy.learn.model_selection import stepwise
+from verticapy.learn.linear_model import LogisticRegression
+from verticapy.tests_new.plotting.conftest import (
+    get_xaxis_label,
+    get_yaxis_label,
+    get_width,
+    get_height,
+)
+
+# Testing variables
+COL_NAME_1 = "age"
+COL_NAME_2 = "fare"
+COL_NAME_3 = "parch"
+COL_NAME_4 = "pclass"
+BY_COL = "survived"
+
+
+class TestHighchartsMachineLearningStepwisePlot:
+    """
+    Testing different attributes of Stepwise plot
+    """
+
+    @pytest.fixture(scope="class")
+    def plot_result(self, schema_loader, titanic_vd):
+        """
+        Create a stepwise regression plot
+        """
+        model = LogisticRegression(
```

Comment:
You may clean up this model at the end.

Reply:
Currently how stepwise is, it cannot train model and plot separately.